### ISSUE TITLE
TypeError: utter_template() got multiple values for argument 'silent_fail'

### ISSUE THREAD

**Issue by JulianGerhard21** [RasaHQ/rasa#4396]
**Rasa version**: 1.3.0

**Rasa X version** (if used & relevant): nr

**Python version**: 3.6.8

**Operating system** (windows, osx, ...): Windows 10

**Issue**:

When a custom form action is triggered, the mentioned error is thrown. Most likely this happens due to the following implementation:

**utter_template in executor.py:**
```
    # noinspection PyUnusedLocal
    def utter_template(
        self,
        template,  # type: Text
        silent_fail=False,  # type: bool
        **kwargs  # type: Any
    ):
```

**forms.py in rasa_sdk**:
```
        for slot in self.required_slots(tracker):
            if self._should_request_slot(tracker, slot):
                logger.debug("Request next slot '{}'".format(slot))
                dispatcher.utter_template(
                    "utter_ask_{}".format(slot),
                    tracker,
                    silent_fail=False,
                    **tracker.slots
                )
                return [SlotSet(REQUESTED_SLOT, slot)]
```
I think this is a classic case of a keyword argument that overwrites a positional argument. I temporarily fixed it by removing the silent_fail from the forms.py.

**Error (including full traceback)**:
```
Traceback (most recent call last):
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask\_compat.py", line 36, in reraise
    raise value
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\flask_cors\decorator.py", line 128, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\rasa_sdk\endpoint.py", line 59, in webhook
    response = executor.run(action_call)
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\rasa_sdk\executor.py", line 252, in run
    events = action(dispatcher, tracker, domain)
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\rasa_sdk\forms.py", line 540, in run
    next_slot_events = self.request_next_slot(dispatcher, temp_tracker, domain)
  File "c:\users\\appdata\local\programs\python\python36\lib\site-packages\rasa_sdk\forms.py", line 388, in request_next_slot
    **tracker.slots
TypeError: utter_template() got multiple values for argument 'silent_fail'
```

**Comment by sara-tagger** [comment]
Thanks for the issue, @akelad will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by tmbo** [comment]
@JulianGerhard21 thanks a lot for the issue report. I just pushed a fix to Rasa SDK to fix this. We removed an unused argument that resulted in some unintended incompatibilities. 

I have readded the argument and released 1.3.1 for Rasa SDK, if you run `pip install -U rasa-sdk` all should be good.
